### ISSUE TITLE
npm audit lists vulnerabilities that auditjs doesn't

### ISSUE THREAD

**Issue by brady305** [sonatype-nexus-community/auditjs#253]
There are cases where npm audit lists vulnerabilities that auditjs doesn't.  If Sonatype thinks those others are false positives, is there an option to include false positives?

**Comment by bhamail** [comment]
If you are asking is it possible for auditjs to include vulnerabilities from npm audit, the short answer is no. ;) Apologies if I'm missing the point.

**Comment by brady305** [comment]
No, what I'm asking is if there is a way to include all vulnerabilities.  In my npm audit run for example, it found that the version of a package called node-forge had two vulnerabilities. In the auditjs run, it didn't find any. So, I just searched for that vulnerability to see if I could find anything about it to see why it would be a false positive. What I found was the opposite. there is a blog post (below link) from Sonatype recommending to upgrade the node-forge package. But auditjs didn't catch it.

New Year, New CVE: a Deep Dive into the ‘node-forge’ (CVE-2022-0122) (sonatype.com)<https://blog.sonatype.com/new-year-new-cve-a-deep-dive-into-the-node-forge-cve-2022-0122>




Thanks,

____________________________

Teri Brady, Lead IT Specialist

IT Applications and Dissemination/Demographic Systems Division

U.S. Census Bureau

O: 301-763-5772 | M: 202-281-5309

census.gov | @uscensusbureau



________________________________

**Comment by madpah** [comment]
This issue has become irrelevant in lieu of AuditJS v5.x that is shipping shortly. AuditJS is being updated to use [Sonatype Guide](https://guide.sonatype.com) as [OSS Index has evolved](https://www.sonatype.com/blog/the-evolution-of-oss-index-in-the-age-of-ai).

Thanks for taking the time to report - hopefully AuditJS v5.x is all you'll ever need anway!
